guide/14-deep-learning/How_MaskRCNN_works.ipynb

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
<p>Perhaps we can add a ground truth image here as well? Right now, it's hard to tell whether these are different buildings or the same. Or perhaps just adding some explanation is enough</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>The link yoou had shared i.e. <a href="https://www.analyticsvidhya.com/blog/2019/07/computer-vision-implementing-mask-r-cnn-image-segmentation/" target="_blank">https://www.analyticsvidhya.com/blog/2019/07/computer-vision-implementing-mask-r-cnn-image-segmentation/</a> also shows comparison without showing ground truth, It looks fine to me. I am editing labels a it which may give more clarity.</p>